frontend/src/components/StartMenuComponent/StartMenuComponent.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {Button, Card, CardContent,} from "@material-ui/core";
+import styled from "styled-components";
+import CardTitle from "../CardTitle/CardTitle";
+import {VerticalSpace} from "../Shared/VerticalSpace";
+import {Centered} from "../Shared/Centered";
+
+export const StartMenuComponent = () => {
+    const startMenuComponent = (
+        <StartMenuCard>
+            <CardContent>
+                <Centered>
+                    <CardTitle title={undefined}/>
+                    <Button variant="contained" color="primary"
+                            onClick={() => console.log("Tournament Button was clicked!")}
+                    >
+                        Turniej
+                    </Button>
+                    <VerticalSpace height="1rem"/>
+                    <Button variant="contained" color="primary"
+                            onClick={() => console.log("Tournament Button was clicked!")}
+                    >
+                        Zawodnicy
+                    </Button>
+                </Centered>
+            </CardContent>
+        </StartMenuCard>
+    );
+    return (
+        startMenuComponent
+    );
+}
+
+const StartMenuCard = styled(Card)({
```

Comment:
Możesz sobie pisać CSSa po prostu w , ale jak wolisz. Składnią jak użyłeś w CardTitle.

```
styled(Card)`
//Tutaj CSS
`
```

Reply:
Nie rozumiem, co chcesz mi tu przekazać.

Reply:
2 różnych składni użyłeś. Tutaj wrzucasz plik JSowy,a wiem, ze Ty lubisz CSS, to możesz pisać jak w CSSie. W CardTitle tak zrobiłeś.

Reply:
A no tak, dzięki!